packages/accounts-base/accounts_server.js

```diff
@@ -742,6 +742,12 @@ Ap._initServerPublications = function () {
 
       const ids = company.childCompanyIds ? [...company.childCompanyIds, company._id] : [company._id];
 
+      let additionalCollections = [];
+
+      if (NC.accountPublications && NC.accountPublications.partners) {
```

Comment:
Maybe piggy back on `Accounts` so we can PR this later into Meteor itself. `Accounts.loginPublish` or similar?